index.html

```diff
@@ -297,12 +297,15 @@ <h1 class="fisto">Contact me</h1><br>
                                    feature that you need built or a 
```

Comment:
- [ ] You have done an excellent job with the accessibility for your portfolio :smiley: but I would suggest updating the `alt` attribute of a few images because you are using the hyphen "-" (e.g., when you mention the icons like "twitter-icon" or "project-image") between the words, and remember that in a common situation this will be read by screen readers and it could be a bit confusing to read. Please, kindly remove the hyphen between the words :wink: 
![image](https://user-images.githubusercontent.com/81522435/145067997-0fc4a346-d17f-433b-b918-f27724bc75ea.png)

- [ ] Also, when you mention the `alt` attribute for decoration items (e.g., a dot), you could use `alt=""` because it's not necessary to mention that is a dot. Please, kindly update the `alt` attribute ONLY for decoration items :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/145068706-debe1518-1a5d-42be-957e-9990610d56a2.png)

- [ ] I would suggest updating the indentation of your could for better practices because it will help you keep your code readable and maintainable :pushpin: :smiley: 
![image](https://user-images.githubusercontent.com/81522435/145068818-0ec2409c-f571-4082-b9dc-369737d618b5.png)